invenio_accounts/views/security.py

```diff
@@ -28,14 +28,8 @@
 @register_menu(
     blueprint,
     "settings.security",
-    # NOTE: Menu item text (icon replaced by a user icon).
-    _(
-        "%(icon)s Security",
-        icon=make_lazy_string(
-            lambda: '<i class="{shield}".format(\
-            shield=current_theme_icons.shield)></i>'
-        ),
-    ),
```

Comment:
My assumption is that it was done like that because you don't need to make changes to the template to make it work, so wherever you define this, it will work. But it is not really separating the concerns, as we want to normally.